stablehlo/dialect/StablehloOps.td

```diff
@@ -398,8 +398,9 @@ def StableHLO_IsFiniteOp: StableHLO_UnaryElementwiseOp<"is_finite", [Pure,
 }
 
 def StableHLO_LogOp: StableHLO_UnaryElementwiseOp<"log",
-    [Pure, HLO_CompatibleOperandsAndResultType], HLO_FpOrComplexTensor> {
-  let summary = "Logarithm operator";
+    [Pure, HLO_CompatibleOperandsAndResultType /*log_c1*/],
+     HLO_FpOrComplexTensor /*log_i1*/> {
+  let summary = "Logarithm operation";
```

Comment:
Can you also change the summary for AbsOp to say "Abs operation"?

Reply:
Looks like it is already "Abs operation"? https://github.com/openxla/stablehlo/blob/main/stablehlo/dialect/StablehloOps.td#L207
Please let me know if I didn't get it.

Reply:
Apologies! I meant ExpOp to say "Exp operation".

Reply:
I see. The summary "Exponential" is align with the op name "exponential", instead of the class name "ExpOp": I think it's better because the readers of Spec will use "stablehlo.exponential" (not "exp") in their StableHLO program.
For log op, yes "Log operation" is good because the op name is indeed "log".
What do you think of it?

Reply:
Got it, understood that this is intentional, and that there are multiple possibilities for this. The checklist recommends "Exp operation", but we can discuss this further after merging this PR. cc @sdasgup3 to drive to a conclusion and update the checklist or the ODS as needed.